config/buildless-serverless/templates/network-policy-allow-egress.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.networkPolicies.enabled }}
 kind: NetworkPolicy
```

Comment:
What do you think about uniting all files containing network policies? In such a case, we could use one `if` for all of them.